webapp/src/main/java/com/box/l10n/mojito/pagerduty/PagerDutyIntegrations.java

```diff
@@ -0,0 +1,41 @@
+package com.box.l10n.mojito.pagerduty;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.stereotype.Component;
+
+@Component
+@DependsOn("pagerDutyIntegrationsConfiguration")
```

Comment:
Is this necessary? Other beans that use configuration don't usually have this afaik

Reply:
I had an issue with the configuration not loading and I was being overly cautious here by ensuring the configuration is loaded before this component is created. Probably best to keep it here.

Reply:
I just find it strange this specific case needs it. Like why is this different from the rest of the cases 🤷‍♂️